### ISSUE TITLE
Keep original video controls (mainly YouTube)

### ISSUE THREAD

**Issue by pintassilgo** [ettoolong/MaximizeVideo#20]
This extension is based on UserScript [Maximize Video](https://greasyfork.org/en/scripts/4870-maximize-video/), right? I'm trying to migrate from the UJS to your addon, but I'm missing original video controls when maximized, mainly in YouTube because of thumbnails preview when mouseover progress bar.

Thank you.

![image](https://user-images.githubusercontent.com/5483864/59641480-17806f80-9138-11e9-9114-e68c44ba9d13.png)

**Comment by ettoolong** [comment]
@pintassilgo No, this extension and [Maximize Video](https://greasyfork.org/en/scripts/4870-maximize-video/), are not associated (even if they have the same name. I think the naming is general).

Basically, this extension not provide original video controls (exclude Twitch, their player is suck, can't be convert to standard HTML5 player).

I am not sure how much time it will take to support original Youtube video controls. Support the original video controls for specific site would be hard and the extension need update frequently (the changes of website may break extension). I don't have so much time to maintain this.

**Comment by ettoolong** [comment]
@pintassilgo
Update v0.0.13,
add new option for this feature:
Maximize Video preferences -> Keep original video controls on Youtube

**Comment by pintassilgo** [comment]
Thank you, but I noticed that click to pause is not working when maximized.
